### PR TITLE
success page

### PR DESCRIPTION
**Pull request by jbubar**
The beginning of the success page. Might need to be reworded. And the boris room link is a blank href rn. it might need to be open before we link to it :D

### CODE REVIEW

File: src/index.scss [hunk 1]
```diff
@@ -33,7 +33,12 @@ html {
     #root {
       height: 100%;
     }
-
+    .height100 {
```
Comment (mackbrowne) [inline src/index.scss]: This is great!  I believe there are bootstrap utility classes you can learn to accomplish these, take a look at [this](https://getbootstrap.com/docs/4.0/utilities/sizing/) to get you started 😁

File: src/App.tsx [hunk 2]
```diff
@@ -49,7 +50,26 @@ export default function App() {
       console.error(message);
     }
   });
-
+  if(captchaSuccess){
```
Comment (mackbrowne) [inline src/App.tsx:52]: Nice 👍 this is perfect !  For what we need, this is more than enough.  

The more 'scalable' option as I'm sure you've already thought would be to use a routing system and make them separate pages with different urls.  There are tradeoffs to both, but ultimately a separate screen with routing is a bit more polished.

File: src/App.tsx [hunk 3]
```diff
@@ -20,13 +20,14 @@ type formValues = {
...
 
 export default function App() {
+  const [captchaSuccess, setCaptchaSuccess] = useState(false)
   const reCapRef = useRef<ReCAPTCHA>();
   const { register, handleSubmit } = useForm<formValues>();
 
   const signUp = handleSubmit(async ({ fullName, email }) => {
     try {
       const token = await reCapRef.current.executeAsync();
-      console.log('token:', token);
```
Comment (mackbrowne) [inline src/App.tsx:29]: 🙌

**Review by mackbrowne — APPROVED** [review]
Everything above looks awesome and ready to deploy! 

The comments I left are just inspiration for future improvements and definitely shouldn't block this code from being merged.

File: src/index.scss (hunk 1 above)
Comment (jbubar) [reply src/index.scss]: Awesome!

Comment (jbubar) [reply src/index.scss]: did this. Thank you!

File: src/App.tsx (hunk 2 above)
Comment (jbubar) [reply src/App.tsx:52]: I think you are right. Lets save that for a refactor

File: src/App.tsx (hunk 3 above)
Comment (jbubar) [reply src/App.tsx:29]: 🙌

**Comment by mackbrowne** [comment]
Epic dude. 👍

**Pull request merged**